storage_backend_s3/README.rst

```diff
@@ -0,0 +1,45 @@
+
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+   :alt: License: LGPL-3
+
+===================
+Storage backend S3
+===================
+
+Add the possibility to store and get data from amazon S3 for your storage backend
```

Comment:
Could you create the folder 'readme' and add some fragments in it? This way, the readme will be updated to OCA guidelines.

Reply:
Added,

Thanks